app/server/src/routes/course/course.route.js

```diff
@@ -0,0 +1,43 @@
+const express = require("express");
+const CourseController = require("./course.controller");
+const { validate } = require("./course.validate");
+const { handleValidation } = require("../../services/validate");
+
+const courseRouter = express.Router();
+
+courseRouter.post(
+  "/createCourse",
+  // validate('createCourse'),
+  // handleValidation,
+  CourseController.createCourse
+);
+
+courseRouter.get(
+  "/getCourses",
+  // validate('getCourses'),
+  // handleValidation,
+  CourseController.getCourses
+);
+
+courseRouter.get(
+  "/getEnrolledCourses",
+  // validate('getEnrolledCourses'),
+  // handleValidation,
+  CourseController.getEnrolledCourses
+);
+
+courseRouter.post(
+  "/createEnrollment",
+  // validate('createEnrollment'),
+  // handleValidation,
```

Comment:
this shouldn't be here... We can create enrollment validate-controller-route .js files and put relevant functions there... and add body parameter checks via validator

Reply:
commented in. Validation functions are not filled.